src/coreclr/gc/gc.cpp

```diff
@@ -25019,7 +25032,22 @@ void gc_heap::check_heap_count ()
 
             // the middle element is the median overhead percentage
             float median_percent_overhead = percent_overhead[1];
-            dprintf (6666, ("median overhead: %d%%", median_percent_overhead));
+
+            // apply exponential smoothing over 3 median_percent_overhead readings
```

Comment:
should say "apply exponential smoothing and use 1/3 for the smoothing factor"? since it's not just over 3 reading.

Reply:
Ok.